DEV_DOCS.md

```diff
@@ -361,6 +361,8 @@ To start a new release (publish the framework packages on NPM) you need:
   * `npm run update-packages-smoke-lock` to update `packages-smoke/package-lock.json` 
   * `npm run version:bump`
   * update version in `package-lock.json` and `packages-smoke/package-lock.json`
+  * update `docs/versions.json`
```

Comment:
what is the process? should we add each new version or only major? we need to state it here

Reply:
Steps added in `cc85005`